grizzly/common/bugzilla.py

```diff
@@ -127,6 +127,12 @@ def load(cls, bug_id: int) -> Optional["BugzillaBug"]:
             return cls(bugzilla.get(bug_id))
         except BugsyException as exc:
             LOG.error("%s", exc.msg)
+            # Access Denied
+            if api_key is None and exc.code == 102:
+                LOG.info(
```

Comment:
Should be `warning`?

Reply:
The exception message is descriptive and printed with LOG.error(). This is a suggestion for local users.